docs/service-catalog/docs/03-04-experimental-features.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Experimental features
+type: Details
+---
+
+Currently Service Catalog requires its own instance of api-server and etcd. This adds additional complexity to the cluster configuration and increases
+maintenance costs. In case of api-server downtime all Service Catalog resources are not available (even simple pod restart can result in such a behaviour).
```

Comment:
```suggestion
In case of api-server downtime, all Service Catalog resources are unavailable. What is more, even the simple Pod restart can result in such behavior.
```